app/main/util/date_filters_validator.py

```diff
@@ -90,6 +84,32 @@ def validate_date_filters(args, browse_consignment=False):
     return date_validation_errors, from_date, to_date, date_filters, error_field
 
 
+def _check_consignment_date_filters(args, date_validation_errors):
+    date_filter_field = args.get("date_filter_field", "")
+
+    # create dictionary for date fields only
+    date_fields = {k: v for k, v in args.items() if "date" in k}
+    if "date_filter_field" in date_fields:
+        date_fields.pop("date_filter_field", None)
+
+    if date_filter_field not in [
+        "date_last_modified",
+        "opening_date",
+    ]:
+        if len([k for k, v in date_fields.items() if len(v) > 0]):
```

Comment:
I think we can get rid of the outer len() since:

- empty list: [] -> len: 0 
boolean value of [] or 0 both False
- non-empty list: ["foo"] -> len: 1 
boolean value of ["foo"] or 1 both True

Reply:
would it make sense to just do 
`date_fields = {k: v for k, v in args.items() if "date" in k and v >0}`
and then not need these extra conditions?

Reply:
updated and removed unwanted conditions